cmd/syncthing/ioprio_linux.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2017 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package main
+
+import "syscall"
+
+const IOPRIO_CLASS_SHIFT = 13
+
+type ioprioClass int
+
+const (
+	IOPRIO_CLASS_RT ioprioClass = iota + 1
+	IOPRIO_CLASS_BE
+	IOPRIO_CLASS_IDLE
+)
+
+const (
+	IOPRIO_WHO_PROCESS = iota + 1
+	IOPRIO_WHO_PGRP
+	IOPRIO_WHO_USER
```

Comment:
Correct go way would be IoprioWhoProcess

Reply:
syscall constants are typically not mangled from their original though, neither in the syscall package nor by us historically

Reply:
fine.